client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue

```diff
@@ -323,6 +306,21 @@ export default {
     updateDrawings (type, data) {
       this.currentPolygons = this.currentPolygons.filter(f => f.geometry.type !== type)
       this.currentPolygons = [...this.currentPolygons, ...JSON.parse(data).features]
+    },
+
+    updateStore (updatedAttributes) {
+      const segment = JSON.parse(JSON.stringify(this.segment))
+
+      segment.attributes = {
+        ...segment.attributes,
+        updatedAttributes
+      }
+
+      const storePayload = {
+        ...segment,
+        id: segment.id
+      }
```

Comment:
with this it would be a bit more compact and we would not need the JSONpars/stringify:
```suggestion
      const storePayload = {
        ...this.segment,
        attributes: {
          ...this.segment.attributes,
          updatedAttributes
        }
        id: segment.id
      }
```